sdcm/nemesis.py

```diff
@@ -128,7 +128,10 @@ def run(self, interval=30):
                     self.log.info('Asked to stop running nemesis')
                     break
             self.target_node.running_nemesis = None
-            time.sleep(interval)
+            if self.termination_event:
+                self.termination_event.wait(timeout=interval)
+            else:
```

Comment:
I think we don't need to sleep for interval. it looks we never got to else:

Reply:
you guys are completely correct, all the logic is now gone